src/ray/common/placement_group.h

```diff
@@ -67,10 +67,18 @@ class PlacementGroupSpecBuilder {
   PlacementGroupSpecBuilder &SetPlacementGroupSpec(
       const PlacementGroupID &placement_group_id, std::string name,
       const std::vector<std::unordered_map<std::string, double>> &bundles,
-      const rpc::PlacementStrategy strategy) {
+      const rpc::PlacementStrategy strategy, const JobID &creator_job_id,
+      const ActorID &creator_actor_id) {
```

Comment:
Get job ID directly from actor ID?

Reply:
I think that's not a good idea because there are cases where actor id is not given. So, we anyway need a job id.